internal/dev_server/api/api.yaml

```diff
@@ -7,9 +7,9 @@ info:
     those flags.
   version: 1.0.0
 servers:
-  - url: "http"
+  - url: "http://localhost:8765/dev"
```

Comment:
This is a somewhat unfortunate change, but this also seems better since the `/dev` prefix isn't repeated everywhere. I needed to remove the path from the routes so that I could use the `dev` prefix to apply the CORS middleware for the these routes.